test/chainsaw/testmetrics/metrics/chainsaw-test.yaml

```diff
@@ -49,18 +49,3 @@ spec:
             # yamllint enable rule:line-length
             check:
               (contains($stdout, '1')): true
-    - name: step-04
```

Comment:
why is this assertion being removed? did we remove the metric?

Reply:
This code was testing the same functionality that is already covered with a different test https://github.com/keptn/lifecycle-toolkit/blob/main/test/chainsaw/integration/podtato-head-application/chainsaw-test.yaml#L19. Also, it makes much more sense to test this behavior on cluster setup where lifecycle-operator is tested than on setup, where only metrics-operator is tested (where this test is executed)